bases/br_capes_bolsas/mobilidade_internacional/table_config.yaml

```diff
@@ -0,0 +1,984 @@
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: grande_area_conhecimento
+      bigquery_type: string
+      description: Grande área do conhecimento
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: nivel_ensino
+      bigquery_type: string
+      description: Nível de Ensino
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations: 'Exemplo: Graduação, doutorado, pós-dourado, etc'
+      is_in_staging: true
+      is_partition: false
+    - name: uf_instituicao_origem
+      bigquery_type: string
+      description: Unidade da Federação da instituição de origem
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id: 
+          table_id: 
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: instituicao_ensino_origem
+      bigquery_type: string
+      description: Nome da instituição de origem
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: instituicao_ensino_principal
+      bigquery_type: string
+      description: Instituição de ensino principal
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: instituicao_ensino
+      bigquery_type: string
+      description: Instituição de ensino
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: deducao_mensalidade_programa_atracao_jovens
+      bigquery_type: float64
+      description: Dedução dos valores pagos de mensalidade (Programa de Atração de Jovens)
+      temporal_coverage:
+          - (1)
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
```

Comment:
esse aqui não é em reais? Se não for, como saberemos qual a moeda desse campo? é tudo pela sigla de moeda de pagamento?

Reply:
Válido pra todos os campos de valor na vdd